### PR TITLE
Adjusting FE rules for microservices

### PR DESCRIPTION
**Pull request by muhammadimranfarooqi**

### CODE REVIEW

**Comment by muhammadimranfarooqi** [comment]
@amaltaro 

This change in FE rule is also required. FYI

File: frontend/backends-k8s-preprod.txt [hunk 1]
```diff
@@ -35,10 +35,10 @@
 ^/auth/complete/das2go(?:/|$) das-server.das.svc.cluster.local
 ^/auth/complete/wmarchive(?:/|$) wmarchive.wma.svc.cluster.local
 ^/auth/complete/exitcodes(?:/|$) exitcodes.http.svc.cluster.local
-^/auth/complete/ms-transferor(?:/|$) vocms0731.cern.ch
-^/auth/complete/ms-monitor(?:/|$) vocms0731.cern.ch
+^/auth/complete/ms-transferor(?:/|$) ms-transferor.dmwm.svc.cluster.local
```
Comment (amaltaro) [inline frontend/backends-k8s-preprod.txt:38]: Imran, sorry for a silly question. But isn't the ms-transferor service called as `reqmgr2ms-transferor` in the kubernetes world? Similar comment for the other two.

Comment (muhammadimranfarooqi) [reply frontend/backends-k8s-preprod.txt:38]: @amaltaro 
No.

https://github.com/dmwm/CMSKubernetes/blob/master/kubernetes/cmsweb/services/reqmgr2ms-monitor.yaml#L4

Comment (amaltaro) [reply frontend/backends-k8s-preprod.txt:38]: I see! Thanks for clarifying it, once again!

**Pull request merged**